specs/WorkerPostMessage.md

```diff
@@ -0,0 +1,682 @@
+    serviceWorker.PostWebMessageAsJson(msg);
+}
+```
+
+# API Details
+## C++
+```
+/// Receives `WebMessageReceived` events.
+[uuid(b366218b-0bb8-58a3-ac33-f40a2235366e), object, pointer_default(unique)]
+interface ICoreWebView2DedicatedWorkerWebMessageReceivedEventHandler : IUnknown {
+  /// Provides the event args for the corresponding event.
+  HRESULT Invoke(
+      [in] ICoreWebView2DedicatedWorker* sender,
+      [in] ICoreWebView2WebMessageReceivedEventArgs* args);
+}
+
+[uuid(66833876-edba-5a60-8508-7da64504a9d2), object, pointer_default(unique)]
+interface ICoreWebView2DedicatedWorker : IUnknown {
+  /// Adds an event handler for the `WebMessageReceived` event.
+  /// Add an event handler for the `WebMessageReceived` event.
+  /// `WebMessageReceived` can be subscribed to, when the
+  /// `ICoreWebView2Settings::IsWebMessageEnabled` setting is set TRUE and the
+  ///  worker runs
+  /// `self.chrome.webview.postMessage`.  The `postMessage` function is
+  /// `void postMessage(object)` where object is any object supported by JSON
+  /// conversion.
+  /// 
+  /// If the worker calls `postMessage` multiple times, the corresponding
+  /// `WebMessageReceived` events are guaranteed to be fired in the same order.
+  HRESULT add_WebMessageReceived(
+      [in] ICoreWebView2DedicatedWorkerWebMessageReceivedEventHandler* eventHandler,
+      [out] EventRegistrationToken* token);
+
+  /// Removes an event handler previously added with `add_WebMessageReceived`.
+  HRESULT remove_WebMessageReceived(
+      [in] EventRegistrationToken token);
+
+
+  /// Post the specific message to this worker.
+  /// The worker receives the message by subscribing to the message event of the
+  /// `self.chrome.webview` of the worker.
+  /// 
+  /// self.chrome.webview.addEventListener('message', handler)
+  /// self.chrome.webview.removeEventListener('message', handler)
+  /// 
+  /// The event args is an instance of `MessageEvent`. The
+  /// `ICoreWebView2Settings::IsWebMessageEnabled` setting must be `TRUE` or
+  /// the web message will not be sent. The data property of the event arg is the
+  /// message string parameter parsed as a JSON string into a JS object.
+  /// The message is delivered asynchronously
+  /// If the worker is terminated or destroyed before the message is posted,
+  /// the message is discarded.
+  HRESULT PostWebMessageAsJson(
+      [in] LPCWSTR messageAsJson
+  );
+
+  /// Posts a message that is a simple string rather than a string
+  /// representation of a JSON object.  This behaves exactly the same
+  /// manner as `PostWebMessageAsJson`, but the `data` property of the event
+  /// arg of the worker's `self.chrome.webview` message is a string with the same
+  /// value as `messageAsString`.  Use this instead of
+  /// `PostWebMessageAsJson` if you want to communicate using simple strings
+  /// rather than JSON objects.
+  HRESULT PostWebMessageAsString(
+      [in] LPCWSTR messageAsString

```

Comment:
nit, param name webMessageAsString to align with CoreWebView2.PostWebMessageAsString?